server/GraknServer.java

```diff
@@ -127,6 +130,16 @@ private Path dataDir() {
         return command.dataDir();
     }
 
+    private void configureAndVerifyJavaVersion() {
+        int javaMajorVersion = Java.getMajorVersion();
```

Comment:
Let's rename the variable to `majorVersion`

Reply:
@lolski addressed in https://github.com/graknlabs/grakn/pull/6251/commits/6958bb9122458f7786ca06b66786c338b46b1240